client/src/store/modules/profile.js

```diff
@@ -57,32 +87,46 @@ export default {
       state.profile = profileData
     },
     updateFollowers (state, profileData) {
-      state.profile.followers = profileData
+      state.profileFollowers = profileData
     },
     updateFollowings (state, profileData) {
-      state.profile.following = profileData
+      state.profileFollowings = profileData
+    },
+    updateCurrentFollowings (state, profileData) {
+      state.currentProfileFollowings = profileData
     }
   },
 
   state: {
     myProfile: {},
     myProfileId: '',
-    profile: {}
+    profile: {},
+    profileFollowers: [],
+    profileFollowings: [],
+    currentProfileFollowings: []
   },
 
   getters: {
     profileGetter (state) {
-      localStorage.lastProfileId = state.profile.profileId
+      // localStorage.lastProfileId = state.profile.profile_id
+      window.localStorage.setItem('lastProfileId', state.profile.profile_id)
       return state.profile
     },
     followingGetter (state) {
-      return state.profile.following
+      return state.profileFollowings
     },
     followersGetter (state) {
-      return state.profile.followers
+      return state.profileFollowers
     },
     myProfile (state) {
       return state.myProfile
+    },
+    followingIdsGetter (state) {
+      let followingIds = []
+      state.currentProfileFollowings.map(function (value, key) {
```

Comment:
Why you need a map here if you pushing ids to another array? Probably better to return ids from a map itself.